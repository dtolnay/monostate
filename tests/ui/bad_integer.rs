use monostate::MustBe;

fn main() {
    let _ = MustBe!(1u15);
}
