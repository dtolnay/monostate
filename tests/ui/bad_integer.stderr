error: unsupported integers suffix `u15`
 --> tests/ui/bad_integer.rs:4:21
  |
4 |     let _ = MustBe!(1u15);
  |                     ^^^^
